drivers/Config.uk

```diff
@@ -1,3 +1,9 @@
+menu "Serial console"
+
+source "$(shell,$(UK_BASE)/support/build/config-submenu.sh -q -o '$(KCONFIG_DIR)/drivers-console.uk' -r '$(KCONFIG_DRIV_BASE)/ukconsole' -l '$(KCONFIG_DRIV_BASE)/ukconsole' -e '$(KCONFIG_EXCLUDEDIRS)')"
```

Comment:
Note that `drivers/ukconsole/Config.uk` must not be provided, otherwise the menu includes things twice.